en/getting-started/first-game/index.md

```diff
@@ -276,72 +579,230 @@ After reading and understanding the capabilities of the __Animation Editor__ cha
     }
     ```
 
-    Click the __Play__ button. When playing, click the left and right mouse buttons, you can see the new jump effect in action:
+6. Click the **Preview** button to see the new jump effect in the open browse by clicking the left or right mouse button.
 
     ![preview with jump](./images/preview-with-jump.gif)
 
-## Upgrading the road
+## Runway upgrade
+
+To make our game more interesting, we need a long runway for the Player to fleet on. Duplicating a lot of Cube nodes is obviously not a good idea, we can automatically create the runway by script.
+
+### Game Manager
+
+Typically, a scene contains various functions and different types of nodes, All those nodes stored in the scene could be taken as the most important data of our game. Occasionally we need to dynamically create, visit, and delete those nodes. Although we can find a node by `find` method in Cocos Creator, actually the `find` method has to enumerate all nodes in the scenes which will cause a low accuracy. In commercial development, we use some single components to manage **some kind of** nodes or **some** nodes data which we call **Manager**.
+
+For example, in a game, we may have many characters, and we may have to create a new character, delete characters who are dead, or query the status of a character, in that case, we can create a class named `ActorManager` to manager all the characters and support the functions mentioned above.
```

Comment:
to manager all -> to manage all

typically , "manager" can't be a verb in English.